extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -179,15 +178,6 @@ const LIFECYCLE_STAGE_TO_ANALYTICS_TRIGGER = {
   'crossDomainIframeLoaded': AnalyticsTrigger.AD_IFRAME_LOADED,
 };
 
-/**
- * @const @enum {string}
- */
-export const NO_SIGNING_EXP = {
-  id: 'a4a-no-signing',
-  control: '21066324',
```

Comment:
If we remove this, where do we track the exp id?

Reply:
This is an RTV experiment so we just compare version numbers.